apps/src/sites/studio/pages/levels/editors/fields/_validation_code.js

```diff
@@ -10,6 +11,9 @@ function initPage() {
     embed.on('click', () => syncValidateWithElements(embed, widgetMode));
     widgetMode.on('click', () => syncValidateWithElements(embed, widgetMode));
   }
+  if (document.getElementById('level_validation_code')) {
```

Comment:
Should we use jquery for this, as we do for `'#level_validation_enabled'` just above?